### DISCUSSION TITLE
要是有收藏功能就更好了

### DISCUSSION THREAD

**Discussion by greenhoie** [share121/inter-knot#1724] [其他]
绳网妥妥的是资源网站啊，宝藏帖子划下去就找不到了

**Comment by share121** [comment]
会开发的

**Reply by greenhoie** [reply]
期待
![屏幕截图 2024-08-12 185401](https://github.com/user-attachments/assets/fb179721-3b69-4b3b-b09c-4bf931d52879)

**Reply by Oaqn** [reply]
转发和“投币”能期待一下吗

**Comment by ChevalierDeSangreal** [comment]
支持！

**Comment by aizai1173** [comment]
顶
